HW04/Ivan_Papushoi/test_methods.rb

```diff
@@ -0,0 +1,48 @@
+require_relative 'ivan_papushoi_hw04.rb'
+require 'rspec/autorun'
+
+RSpec.describe Array do
+  it 'check my_select return' do
+    array = [1, 2, 3, 4, 5, 6]
+    term = ->(x) { x.odd? }
+    expectation = [1, 3, 5]
+    expect(array.my_select { |x| term[x] }).to eq(expectation)
+  end
+
+  it 'compare select and my_select' do
+    array = Array.new(20) { rand(-100..100) }
+    term = ->(x) { x.negative? }
+    expect(array.my_select { |x| term[x] }).to eq(array.select { |x| term[x] })
+  end
+end
+
+RSpec.describe Array do
```

Comment:
You don't need to create the same block many times would be better to go by this way 
```suggestion
RSpec.describe Array do
  describe '#my_each' do
  ...
  describe '#my_select' do
  ...
  
```

Reply:
I did it because rubocop warn me "too much lines in block Array", but now I know how to disable every kind of cops :imp: